pkg/standalone/standalone.go

```diff
@@ -48,8 +49,64 @@ const (
 	DaprRedisContainerName = "dapr_redis"
 )
 
+func isInstallationRequired(installLocation, requestedVersion string) bool {
+	destDir := daprDefaultLinuxAndMacInstallPath
+	if installLocation == "" {
+		if runtime.GOOS == daprWindowsOS {
+			destDir = daprDefaultWindowsInstallPath
+		}
+	}
+
+	daprdBinaryPath := filepath.Join(destDir, daprRuntimeFilePrefix) //e.g. /usr/local/bin/daprd or c:\\daprd
+
+	//First time install?
+
+	_, err := os.Stat(daprdBinaryPath)
+	if os.IsNotExist(err) {
+		return true
+	}
+
+	var msg string
+
+	//what's the installed version?
+	v, err := utils.RunCmdAndWait(daprdBinaryPath, "--version")
+	if err != nil {
+		msg = fmt.Sprintf("unable to determine installed Dapr version at %s", destDir)
+		print.FailureStatusEvent(os.Stdout, msg)
+		return false
+	}
+	installedVersion := strings.TrimSpace(v)
+
+	//"latest" version requested. need to check the corresponding version
+	if requestedVersion == daprLatestVersion {
+		latestVersion, err := getLatestRelease(daprGitHubOrg, daprGitHubRepo)
```

Comment:
We moved daprGitHubOrg and daprGitHubRepo constants to pkg/version/version.go. please import version pkg.

```
import cli_ver "github.com/dapr/cli/pkg/version"
```

Reply:
done!